BTCPayServer/wwwroot/main/rtl.css

```
@@ -0,0 +1,226 @@
+/* what must be RTLed */
+body,
+#NotificationsDropdown,
+#StoreSelectorToggle,
+.accordion-button,
+.navbar-nav,
+#globalNav .nav-link,
+#globalNav .globalSearch-shell
+{
+    direction: rtl;
+}
+/* what must not be RTLed like the pay button code box...*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)